### PR TITLE
Add zapier.com

### PR DESCRIPTION
**Pull request by handrus**

### CODE REVIEW

**Comment by handrus** [comment]
Since I lost #352 I just added notes and opened a new PR

File: _data/sites.json [hunk 1]
```diff
@@ -8070,6 +8070,17 @@
             "yummly.com"
         ]
     },
+    
+    {
+        "name": "Zapier",
+        "url": "https://zapier.com/app/settings/delete",
+        "difficulty": "easy",
+        "notes": "Got to the URL, confirm your email. You'll be redirected to the login page. Confirm your email again and enter \"DELETE\" (all caps) In the \"Confirm you want to delete your account\" field.",
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion
        "notes": "Go to the URL, confirm your email. You'll be redirected to the login page. Confirm your email again and enter \"DELETE\" (all caps) In the \"Confirm you want to delete your account\" field.",
```
Minor typo

File: _data/sites.json [hunk 2]
```diff
@@ -8070,6 +8070,17 @@
             "yummly.com"
         ]
     },
+
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion

```
Remove trailing spaces to make the CI happy

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**